Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -1107,14 +1107,18 @@ void PaintableBox::resolve_paint_properties()
 
     // Border radii
     CSSPixelRect const border_rect { 0, 0, border_box_width(), border_box_height() };
```

Comment:
`border_rect` can be moved inside your new `if (computed_values.has_noninitial_border_radii()) {}` block.